src/NServiceBus.AzureFunctions.StorageQueues/StorageQueueTriggeredEndpointConfiguration.cs

```diff
@@ -37,6 +37,7 @@ public StorageQueueTriggeredEndpointConfiguration(string endpointName, string co
             recoverability.Delayed(settings => settings.NumberOfRetries(0));
 
             Transport.DelayedDelivery().DisableTimeoutManager();
+            Transport.DelayedDelivery().DisableDelayedDelivery();
```

Comment:
`DisableDelayedDelivery` also disables the timeout manager, so we can remove the first line

```suggestion
            Transport.DelayedDelivery().DisableDelayedDelivery();
```